packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/RestaurantDish.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div :class="[$style['c-restaurantCard-dish']]">
+        <p :class="[$style['c-restaurantCard-dish-description']]">
+            <span
+                data-test-id="restaurant-dish-name"
+                :class="[$style['c-restaurantCard-dish-name']]">
+                {{ name }}
+            </span>
+            <span
+                v-if="calories"
+                :class="[$style['c-restaurantCard-dish-nutritionalInfo']]">
+                <span
+                    data-test-id="restaurant-dish-calories"
+                    :class="[$style['c-restaurantCard-dish-nutritionalInfo-item']]">
+                    {{ calories }}
+                </span>
+                <span
+                    v-if="portion"
+                    data-test-id="restaurant-dish-portion"
+                    :class="[$style['c-restaurantCard-dish-nutritionalInfo-item']]">
+                    {{ portion }}
+                </span>
+            </span>
+        </p>
+        <p
+            v-if="price"
+            data-test-id="restaurant-dish-price"
+            :class="[$style['c-restaurantCard-dish-price']]">
+            {{ price }}
+        </p>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'RestaurantDish',
+    props: {
+        name: {
+            type: String,
+            default: ''
+        },
+        /**
+         * String representation of how much the dish costs, including currency. E.g. £2.79
+         */
+        price: {
+            type: String,
+            default: null
+        },
+        /**
+         * String representation of the calorie count of the dish. Includes unit. E.g. 1000kcal
+         */
+        calories: {
+            type: String,
+            default: null
+        },
+        /**
+         * String representation of the portion size of the dish. I.e. '2 Servings'
+         */
+        portion: {
+            type: String,
+            default: null
+        }
+    }
+};
+</script>
+
+<style lang="scss" module>
+.c-restaurantCard-dish {
```

Comment:
nit: indentation in this declaration doesn't match the rest below

Reply:
oops!